common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1674,6 +1674,38 @@
       "Function <funcName> does not implement ScalarFunction or AggregateFunction."
     ]
   },
+  "INVALID_UPDATE_FIELD" : {
+    "message" : [
+      "Cannot update <table> field <fieldName> type:"
+    ],
+    "subClass" : {
+      "ARRAY_TYPE_UNSUPPORTED" : {
```

Comment:
Let's make sub-class names shorter like `ARRAY_TYPE`. In any case, users will observe `CANNOT_UPDATE_FIELD .ARRAY_TYPE`